rules/bitstreams.bzl

```diff
@@ -95,3 +95,229 @@ bitstreams_repo = repository_rule(
     # context, see <https://github.com/lowRISC/opentitan/issues/16832>.
     configure = True,
 )
+
+# Bitstream manifests follow a schema written in the JSON Schema language, and
+# all file paths in the manifest are relative to the manifest's directory.
+# The tools provided to operate on bitstream cache entries currently create
+# directories with the following layout:
```

Comment:
FYI: The tools generate entries with this layout, but they do not _expect_ this directory layout for inputs (unless I made a mistake, heh).